### ISSUE TITLE
Update to Jekyll 4.2.0

### ISSUE THREAD

**Issue by breun** [envygeeks/jekyll-docker#297]
Jekyll 4.2.0 was released on December 14: https://jekyllrb.com/news/2020/12/14/jekyll-4-2-0-released/

**Comment by dgw** [comment]
I see there's [a 4.2.0 tag](https://hub.docker.com/layers/jekyll/jekyll/4.2.0/images/sha256-de110e0fdde342de0cb6fd5c63f64934b6ce25d0dd826c67113b3a5a3370784e?context=explore) on Docker Hub.

The aliases are a bit sparse compared to the 3.x series, though. Moving a site at work from 3.8 to 4.2 didn't go well on first build because I assumed there would be a `4.2` tag, just like there's a `3.8` tag. There's not. It looks like `4.0` points to the same image as `4.2.0`, which is unexpected. I don't want to pin us to 4.2.0 only, but don't want to pick `4.0` or `4` as the tag and then get unexpected breaking changes from a future 4.x point release.

**tl;dr:** Can we get a `4.2` tag that points to `4.2.0` now, and gets updated as maintenance releases come out?

**Comment by breun** [comment]
I'm closing this ticket, because the image was updated to Jekyll 4.2.0 via https://github.com/envygeeks/jekyll-docker/commit/575057ab5f14055e161b871e75f8295afd571aa9.

**Comment by dgw** [comment]
There is 4.2.1 now, and a 3.9.x branch as well that's been skipped entirely in the Docker tags.

But mostly, I was hoping to see more range tags like `4.2` (pointing to 4.2.0 now, and then to 4.2.1 etc. when patches are released).
